examples/playground14/app/api/instrumentation/route.ts

```diff
@@ -0,0 +1,10 @@
+import { NextResponse } from "next/server";
+
+export const dynamic = "force-dynamic";
+
+export function GET() {
+  return NextResponse.json({
+    "nodejs-instrumentation-setup": globalThis["__NODEJS_INSTRUMENTATION_SETUP"] ?? "undefined",
```

Comment:
is `?? "undefined"` really needed?

Reply:
no, but without it you would not get the field at all, for making this more self-explanatory I find it nice to clearly include "undefined" instead, but if you prefer the other version I'll switch to it